helm/vertical-pod-autoscaler-app/templates/crd-patch/crd-np.yaml

```diff
@@ -1,7 +1,7 @@
 kind: NetworkPolicy
 apiVersion: networking.k8s.io/v1
 metadata:
-  name: {{ include "vpa.fullname" . }}-crd-patch
```

Comment:
No crd patch so no netpol right?

Reply:
Should we delete the entire crd-patch folder ?